src/node_file.h

```diff
@@ -13,6 +14,8 @@ namespace node {
 using v8::Context;
 using v8::FunctionCallbackInfo;
 using v8::HandleScope;
+using v8::Integer;
+using v8::Isolate;
```

Comment:
We should really avoid adding `using` statements to header files – yes, `node_file.h` doesn’t follow best practices here to begin with, but let’s not make this worse…